release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -179,6 +179,18 @@ With this release, you can view the ingress network flows for {product-title} se
 
 For more information, see xref:../installing/install_config/configuring-firewall.html#network-flow-matrix_configuring-firewall[{product-title} network flow matrix].
 
+[id="ocp-4-16-networking-pathing-existing-dual-stack"]
+==== Patching an existing dual-stack network
+
+With this release, you can add IPv6 virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster by patching the cluster's infrastructure.
```

Comment:
```suggestion
With this release, you can add IPv6 virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster by patching the cluster infrastructure.
```
Don't need the possessive here. https://spaces.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Possessives+and+apostrophes

Reply:
:heavy_check_mark: